service/src/test/resources/ehr/mapper/diagnosticreport/diagnostic-report-with-coded-diagnosis.xml

```diff
@@ -9,6 +9,7 @@
             <center nullFlavor="NI"/>
         </effectiveTime>
         <availabilityTime value="20100225154100"/>
+            <!-- Mapped Specimen with id: Specimen/Default-1 -->
```

Comment:
Should we still be outputting a comment / placeholder value here?

Reply:
Specimen is mapped in a different class and it's mocked in DiagnosticReport tests.